### PR TITLE
[Term Entry] Python Math Module: math.acos()

### LINKED ISSUE #5076: [Topic Entry] Python math-module math.acos()
### Reason/inspiration (optional)

We would like a new term entry on the `math.acos()` method of the [`math-module`](https://www.codecademy.com/resources/docs/python/math-module) concept in Python. The entry should go in a new file under `docs/content/python/concepts/math-module/terms/math-acos/math-acos.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Topic Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by raheep06**
### Issue Solved
Closes #5076 

### Type of Change

- Adding a new entry

### Checklist

- ✅ All writings are my own.
- ✅ My entry follows the Codecademy Docs style guide.
- ✅ My changes generate no new warnings.
- ✅ I have performed a self-review of my own writing and code.
- ✅ I have checked my entry and corrected any misspellings.
- ✅ I have made corresponding changes to the documentation if needed.
- ✅ I have confirmed my changes are not being pushed from my forked `main` branch.
- ✅ I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- ✅ I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/math-module/terms/math-acos/math-acos.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
```
Comment (mamtawardhani) [inline content/python/concepts/math-module/terms/math-acos/math-acos.md:13]:
```suggestion
---
Title: 'math.acos()'
Description: 'Returns the arc cosine (inverse cosine) of a number.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Arithmetic'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

```

Comment (mamtawardhani) [reply content/python/concepts/math-module/terms/math-acos/math-acos.md:13]: The description should be the function, and the tags should be alphabetically arranged

File: content/python/concepts/math-module/terms/math-acos/math-acos.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The **`math.acos()`** function returns the inverse cosine (in radians) of a value between -1 and 1.
```
Comment (mamtawardhani) [inline content/python/concepts/math-module/terms/math-acos/math-acos.md:15]:
```suggestion
In Python, the **`math.acos()`** function returns the arc cosine (inverse cosine) in radians of a value between `-1` and `1`.
```

File: content/python/concepts/math-module/terms/math-acos/math-acos.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Syntax
+
+```pseudo
+math.acos(n)
```
Comment (mamtawardhani) [inline content/python/concepts/math-module/terms/math-acos/math-acos.md:20]:
```suggestion
math.acos(x)

```

File: content/python/concepts/math-module/terms/math-acos/math-acos.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+```
+
```
Comment (mamtawardhani) [inline content/python/concepts/math-module/terms/math-acos/math-acos.md:22]:
```suggestion

- `x`: A number between `-1` and `1` representing the cosine value for which the arc cosine is to be calculated.

```

Comment (mamtawardhani) [reply content/python/concepts/math-module/terms/math-acos/math-acos.md:22]: The parameter needs to be described, feel free to change the description this is just for reference

File: content/python/concepts/math-module/terms/math-acos/math-acos.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+The `math.acos()` function returns a numeric value between 0 and π, which represents the inverse cosine of the value `n`.
```
Comment (mamtawardhani) [inline content/python/concepts/math-module/terms/math-acos/math-acos.md:23]:
```suggestion
The `math.acos()` function returns a numeric value between 0 and π, which represents the inverse cosine of the value `x`.

```

File: content/python/concepts/math-module/terms/math-acos/math-acos.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+## Example
+
+Use `math.acos()` to return the inverse cosine of a numeric value between 0 and π:
```
Comment (mamtawardhani) [inline content/python/concepts/math-module/terms/math-acos/math-acos.md:27]:
```suggestion
The example below demonstrates the use of `math.acos()` function to return the inverse cosine of a numeric value:

```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @raheep06 thank you for contributing to the Codecademy Docs, the entry looks nicely written! 😃 

I've suggested a few changes, could you please review and modify them at your earliest convenience? Thank you! 😄

**Review by raheep06 — COMMENTED** [review]
Updated all sections based on feedback. Thank you

**Review by mamtawardhani — APPROVED** [review]
Well done @raheep06 😃 

The entry looks good for a second round of review 🚀

**Review by Sriparno08 — APPROVED** [review]
Looks good, @raheep06!

**Pull request merged**